R/manual_layer_group_deregister.R

```diff
@@ -0,0 +1,64 @@
+
+##' De-register a 'Group' object recursively
+##'
+##' This function de-registers a 'Group' object, the 'Group' objects therein
+##' as well as any 'arrays.
```

Comment:
`'arrays` -> `'arrays'` ?

Reply:
Thanks -- will fix to no apostrophes:   "as well as any arrays."